### PR TITLE
39: Generate theme.

### PR DESCRIPTION
**Pull request by josue2591**
**Steps to test**
- Check if the new module manatime_theme_activator was added to the settings.php file.
- Verify the theme is activated automatically.
- Check if the ruby files were deleted.
- Review the manatime_theme_activator code, was created fine.

[Issue link](https://github.com/ManatiCR/manatime/issues/39)

### CODE REVIEW

File: modules/custom/manatime_theme_activator/manatime_theme_activator.module [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?php
+/**
+ * @file
+ * Manatime Theme module functions.
```
Comment (kporras07) [inline modules/custom/manatime_theme_activator/manatime_theme_activator.module]: Manatime Thme Activator module functions.

File: themes/custom/manatime_theme/.editorconfig [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# http://editorconfig.org
```
Comment (kporras07) [inline themes/custom/manatime_theme/.editorconfig]: Delete this file. It's integrated in root .editorconfig.

File: themes/custom/manatime_theme/.gitignore [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.sass-cache
```
Comment (kporras07) [inline themes/custom/manatime_theme/.gitignore]: Delete this file. It's integrated in root .gitignore

File: themes/custom/manatime_theme/.jshintrc [hunk 4]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (kporras07) [inline themes/custom/manatime_theme/.jshintrc]: Delete this file.

File: themes/custom/manatime_theme/.yo-rc.json [hunk 5]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (kporras07) [inline themes/custom/manatime_theme/.yo-rc.json]: Delete this file.

File: themes/custom/manatime_theme/gulpfile.js [hunk 6]
```diff
@@ -0,0 +1,83 @@
+'use strict';
```
Comment (kporras07) [inline themes/custom/manatime_theme/gulpfile.js]: Delete this file. Instead, copy gulp-tasks/theme.js from retopais to root gulp-tasks folder and change appropiate values.

File: themes/custom/manatime_theme/js/script.js [hunk 7]
```diff
@@ -0,0 +1,26 @@
+/**
```
Comment (kporras07) [inline themes/custom/manatime_theme/js/script.js]: Delete unnecessary stuff from this file.

File: themes/custom/manatime_theme/package.json [hunk 8]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (kporras07) [inline themes/custom/manatime_theme/package.json]: Delete this file. It's going to be integrated in root package.json.

**Comment by kporras07** [comment]
a7fc014bf8488b37d60ed7dc6c76acbff5d3fb7491ff3cbd302a4a9a4743ef14
 with the branch name :grin:

**Comment by kporras07** [comment]
Just add what I sent you through chat and this will Pass Code Review. Then, I'll do functional review.

**Comment by kporras07** [comment]
Missing aurora in makefile

File: themes/custom/manatime_theme/js/script.js (hunk 7 above)
Comment (cienvaras) [reply themes/custom/manatime_theme/js/script.js]: Delete this file, there's no guaranty we will use it.

**Comment by cienvaras** [comment]
:shipit: !! (assuming clean wercker build).

**Pull request merged**